org/mozilla/jss/netscape/security/pkcs/PKCS8Key.java

```diff
@@ -196,8 +197,12 @@ public static PKCS8Key buildPKCS8Key(AlgorithmId algid, byte[] key)
         } catch (ClassNotFoundException e) {
         } catch (InstantiationException e) {
         } catch (IllegalAccessException e) {
-            // this should not happen.
-            throw new IOException(classname + " [internal error]");
+            throw new IOException("IllegalAccessException : " +
+                      e.getMessage());
```

Comment:
I can fix this up for you, but something @edewata showed me is that the Exception constructor can take a `Throwable` so you preserve the original exception and its stack trace (`Exception(String message, Throwable cause)`):

```Java
throw new IOException("IllegalAccessException : " + e.getMessage(), e);
```